example_configs/nextcloud.md

```diff
@@ -5,24 +5,22 @@
 This example is using following users & groups in lldap :
 
 * A technical user (ex: `ro_admin`), member of `lldap_strict_readonly` or `lldap_password_manager`
-* A catch-all group called `nextcloud_users`. Note: This is _not_ the same as the `Users` designation in the LLDAP Web UI. This must be a whole new group that you've created and called `nextcloud_users`. 
+* A catch-all group called `nextcloud_users`.
 * Members of `nextcloud_users` group will be authorized to log in Nextcloud.
 * Some "application" groups, let's say `friends` and `family`: users in Nextcloud will be able to share files and view people in dynamic lists only to members of their own group(s).
-* Users in 'family' and 'friends' should also be users in 'nextcloud_users' group!  
+* Users in `family` and `friends` should also be users in `nextcloud_users` group!  
 
 If you plan on following this tutorial line-by-line, you will now have the following:
 * 6 groups:
-    1. `users`
+    1. `nextcloud_users`
     2. `family`
     3. `friends`
     4. `lldap_strict_readonly`
     5. `lldap_password_manager`
-    6. 'lldap_admin'
-    7. `admin`
-* 1 admin user in one or both of the following groups:
+    6. `admin`
```

Comment:
oh my 🫣 you must hate this pr. hopefully the last error! In my original setup, I had an admin group (which is pointless since `lldap_admin exists`) and completely overlooked it when I moved from my gist to here. Thanks again.